geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/IngestCommand.scala

```diff
@@ -131,11 +129,12 @@ trait IngestCommand[DS <: DataStore] extends DataStoreCommand[DS] with Interacti
       libjarsFile, libjarsPaths, params.threads)
   }
 
-  private def ensureSameFs(prefixes: Seq[String]): Unit = {
-    prefixes.foreach { pre =>
-      if (params.files.exists(_.toLowerCase.startsWith(s"$pre://")) &&
-        !params.files.forall(_.toLowerCase.startsWith(s"$pre://"))) {
-        throw new ParameterException(s"Files must all be on the same file system: ($pre) or all be local")
+  private def ensureSameFs(): Unit = {
+    if (params.files.exists(PathUtils.isRemote)) {
+      // If we have a remote file, make sure they are all the same FS
+      val prefix = params.files.head.split(":")(0)
```

Comment:
i think this needs to be lower-cased to match the check on the next line. Actually, this check may not be entirely robust, as something like 's3' would match 's3a'...